RELEASE_NOTES.md

```diff
@@ -1,3 +1,7 @@
+### Version 1.9.26 (September 24th, 2015)
+* Skipped 1.9.25
```

Comment:
I'm ok with this being here, but I sort of feel like there might be a better way to say that a version was skipped.